src/v2/Apps/Order/Routes/Respond/index.tsx

```diff
@@ -277,20 +277,24 @@ export class RespondRoute extends Component<RespondProps, RespondState> {
                         onChange={offerValue => this.setState({ offerValue })}
                         onFocus={this.onOfferInputFocus.bind(this)}
                       />
-                      <Spacer mb={0.5} />
-                      <RevealButton
-                        align="left"
-                        buttonLabel="Add note to seller"
```

Comment:
Realized when collectors submit a counteroffer, they can also include a note. I think it still makes sense but I'm curious about user's feedback.

Reply:
Still makes sense to show this offer note for the response view? And keep it hidden for the initial offer view?

Reply:
Oh sorry I meant still makes sense to hide it. 😅

Reply:
Ah yeah I understand now.